dev/com.ibm.ws.concurrent.persistent_fat_auto_pollinterval/fat/src/com/ibm/ws/concurrent/persistent/fat/autonomicalpolling1serv/AutonomicalPolling1ServerTest.java

```diff
@@ -512,7 +515,7 @@ public void testAdd3Remove2Add4PersistentExecs() throws Exception {
 		Thread.sleep(3000);
 		
 		// Add 4 persistent executors.
-		for (int i = 0; i < 4; i++) {
+		for (int i = 1; i < 5; i++) {
```

Comment:
I think this needs to be `i = 3; i < 7;` so that we don't override 1 & 2, which were previously disabled.